### PR TITLE
Update QUEST.tsv [Lines 501-1000]

### PR DESCRIPTION
**Pull request by ghost**
Major rewrite.
I've done this until line 2275 which I'll submit in batches through other pull requests in the next few days after I do some final checks.

Open to feedback.

### CODE REVIEW

File: QUEST.tsv [hunk 1]
```diff
@@ -498,20 +498,20 @@ QUEST_20150317_000497	The harder you try to find this, the stronger you become,
 QUEST_20150317_000498	Aide de camp Selina
 QUEST_20150317_000499	Our days to stay self-sufficient is growing. {nl}We even joke about plowing a farm ourselves,  {nl}but it's not becoming a joke anymore.
 QUEST_20150317_000500	Watcher
-QUEST_20150317_000501	If it isn't the one who made me release the Saule goddess. {Nl}What are you putting your life on the line for?
-QUEST_20150317_000502	There is no need for more of your work. {nl}You should follow the others by immersing yourself in peace {nl}and praying the Goddess will grant you your wishes.
-QUEST_20150317_000503	I warned you, but you just won't listen.
+QUEST_20150317_000501	If it isn't you, the one who made me release the Saule Goddess. {nl}Then what are you putting your life on the line for?
```
Comment (Sunaries) [inline QUEST.tsv]: If it isn't you, the one who made me release the Saule Goddess

↓

If it isn't you, the one who made me release Goddess Saule. 

Sounds better.

File: QUEST.tsv [hunk 2]
```diff
@@ -773,99 +773,99 @@ QUEST_20150317_000772	There are some old pots among the new pots. {nl}Which one
...
-QUEST_20150317_000838	Huh…{nl}Who would've thought I'd be swayed by humans…
-QUEST_20150317_000839	I shall kill you in the most painful way!
+QUEST_20150317_000837	Dear Revelator. {nl}Listen to the interpretations of the revelation.
+QUEST_20150317_000838	Huh… {nl}Who'd've thought I'd be swayed by humans…
+QUEST_20150317_000839	Your death shall be painful!
 QUEST_20150317_000840	Soldier Recruitment Notice
-QUEST_20150317_000841	We want you.{Nl}Protect your family and your kingdom.{nl} {nl} - Klaipeda Recruit
-QUEST_20150317_000842	We are seeking brave young people who will bring order to this chaotic time. {nl} You are what the world wants. {nl} - Klaipeda Recruit
-QUEST_20150317_000843	Honor.{nl}This is what your uniform will be made of. {nl} {nl} - Klaipeda Recruit{nl}
+QUEST_20150317_000841	We want you! {nl}Protect your family and your kingdom. {nl} - Klaipeda Lesion
```
Comment (Sunaries) [inline QUEST.tsv]: Why did you change this to Lesion? It should be: Klaipeda Recruiting Station/Center

File: QUEST.tsv [hunk 3]
```diff
@@ -773,99 +773,99 @@ QUEST_20150317_000772	There are some old pots among the new pots. {nl}Which one
...
+QUEST_20150317_000842	We are seeking brave young men who will bring order to this chaotic time. {nl}You are what the world wants. {nl} - Klaipeda Lesion
+QUEST_20150317_000843	Honor. {nl}That is what your uniform will be made of. {nl} - Klaipeda Lesion
```
Comment (Sunaries) [inline QUEST.tsv]: Why did you change this to Lesion? It should be: Klaipeda Recruiting Station/Center

File: QUEST.tsv [hunk 4]
```diff
@@ -773,99 +773,99 @@ QUEST_20150317_000772	There are some old pots among the new pots. {nl}Which one
...
-QUEST_20150317_000839	I shall kill you in the most painful way!
+QUEST_20150317_000837	Dear Revelator. {nl}Listen to the interpretations of the revelation.
+QUEST_20150317_000838	Huh… {nl}Who'd've thought I'd be swayed by humans…
+QUEST_20150317_000839	Your death shall be painful!
 QUEST_20150317_000840	Soldier Recruitment Notice
-QUEST_20150317_000841	We want you.{Nl}Protect your family and your kingdom.{nl} {nl} - Klaipeda Recruit
-QUEST_20150317_000842	We are seeking brave young people who will bring order to this chaotic time. {nl} You are what the world wants. {nl} - Klaipeda Recruit
-QUEST_20150317_000843	Honor.{nl}This is what your uniform will be made of. {nl} {nl} - Klaipeda Recruit{nl}
+QUEST_20150317_000841	We want you! {nl}Protect your family and your kingdom. {nl} - Klaipeda Lesion
+QUEST_20150317_000842	We are seeking brave young men who will bring order to this chaotic time. {nl}You are what the world wants. {nl} - Klaipeda Lesion
```
Comment (Sunaries) [inline QUEST.tsv]: Why did you change this to Lesion? It should be: Klaipeda Recruiting Station/Center

File: QUEST.tsv [hunk 5]
```diff
@@ -946,58 +946,58 @@ QUEST_20150317_000945	Eating more would be rather risky.
...
-QUEST_20150317_000998	I've been throwing up a lot lately but it seems to be getting better now.  {Nl}Tomorrow I'm going to go see Mr. Gordon to get some medicinal herbs. {Nl} My little sister still hasn't come back yet. When I go to her room, I find no clothes. Even Mr. Man is getting worried.
-QUEST_20150317_000999	X month the 17th {nl} - {nl} The village was eerily quiet today.  {Nl}There are no signs of the people who went earlier to get herbs from Mr. Gordon.
-QUEST_20150317_001000	X month the 23rd {nl} - {nl} This might be the last time I get to write in this diary. {Nl} Liliana, you are the only family I have left in this world… {nl} Please be alive somewhere… {Nl}
+QUEST_20150317_000989	X Month, the 4th {nl}- {nl}There's been very little traffic coming to town from the capital. {nl}When one family moved in, the whole village threw a welcome party. {nl}After all, how many outsiders do we get to meet these days?
```
Comment (Sunaries) [inline QUEST.tsv]: In the original it says X월. If it were 1월 it would be January, or it would be October if it said 10월.
I suggest changing it either to: _X, the 4th._ or _The 4th of X_. 

Remember to edit the other lines as well for consistency. Thanks.

File: QUEST.tsv [hunk 6]
```diff
@@ -522,210 +522,210 @@ QUEST_20150317_000521	Seems to be asleep.
 QUEST_20150317_000522	Enraged Owl Statue
 QUEST_20150317_000523	No responses.
 QUEST_20150317_000524	Regretful Merchant's Soul
-QUEST_20150317_000525	If you don't have money, you're no different than a dead man.
-QUEST_20150317_000526	I should've listened to my wife. {nl}She said it's bad luck to be friends with scholars… {nl}I think I'm really going out of business!
+QUEST_20150317_000525	If you don't have money, you're no different from a dead man.
+QUEST_20150317_000526	I should've listened to my wife. {nl}She says it's bad luck to be friends with scholars… {nl}I'm really going out of business!
```
Comment (ttgmichael) [inline QUEST.tsv]: I think the past tense was ok here? The wife stated it was bad luck. It's not like she's saying it now.

File: QUEST.tsv [hunk 7]
```diff
@@ -522,210 +522,210 @@ QUEST_20150317_000521	Seems to be asleep.
...
+QUEST_20150317_000555	One day, I will find out. {nl}Even if that entails my meeting the Goddess.
 QUEST_20150317_000556	I don't know what to do if you are like that…
 QUEST_20150317_000557	Crystal Mine 2nd Lot is closed until further notice due to safety reasons. {nl} - Klaipeda Mines Union
 QUEST_20150317_000558	Business Guide
 QUEST_20150317_000559	Welcome! This is the miner's resting hall. {nl}Warm potatoes and drinks have been prepared for you. {nl}{nl}Business hours: same as mining time - may change seasonally.
 QUEST_20150317_000560	Notice
-QUEST_20150317_000561	This is the Crystal Mines. {Nl}Infractions will be given and you will be forced to leave if you do not observe the following:
-QUEST_20150317_000562	1. Always wear a helmet and protective gear {nl}2. Do not enter restricted areas {nl}3. Never carry flamable objects {nl}4. If you see monsters, do not fight and file a report immediately.
+QUEST_20150317_000561	This is the Crystal Mine. {nl}You will be punished and forced to leave if you do not observe the following:
```
Comment (ttgmichael) [inline QUEST.tsv]: This is the Crystal Mine. Violators of the rules below will be punished and forced to leave.

note: this is a sign in the game

File: QUEST.tsv [hunk 8]
```diff
@@ -498,20 +498,20 @@ QUEST_20150317_000497	The harder you try to find this, the stronger you become,
...
-QUEST_20150317_000501	If it isn't the one who made me release the Saule goddess. {Nl}What are you putting your life on the line for?
-QUEST_20150317_000502	There is no need for more of your work. {nl}You should follow the others by immersing yourself in peace {nl}and praying the Goddess will grant you your wishes.
-QUEST_20150317_000503	I warned you, but you just won't listen.
+QUEST_20150317_000501	If it isn't you, the one who made me release Goddess Saule. {nl}Then what are you putting your life on the line for?
+QUEST_20150317_000502	There's no need for you to work anymore. {nl}You should follow the others' example and immerse yourself in peace, {nl}and pray the Goddess will grant you your wishes.
+QUEST_20150317_000503	I warned you but you just wouldn't listen.
 QUEST_20150317_000504	You shall pay for your choice.
 QUEST_20150317_000505	I underestimated you.
-QUEST_20150317_000506	But do not think that this is over. {nl}I'll be watching from the Fallen Forest.
+QUEST_20150317_000506	This isn't over! {nl}I'll be watching from the Fallen Worry Forest.
```
Comment (terrapin47) [inline QUEST.tsv:506]: ( First sentence ) There doesn't seem to be a difference in what is conveyed here, just a change of tone.

File: QUEST.tsv [hunk 9]
```diff
@@ -522,210 +522,210 @@ QUEST_20150317_000521	Seems to be asleep.
...
 QUEST_20150317_000647	Pyromancers use the blessings of Gabija, the Goddess of Fire, as their main force.
 QUEST_20150317_000648	It differs on the type of element, and depends on the grace of Goddess Gabija, {nl}but some elements may ignore oxidation.
-QUEST_20150317_000649	For this, you will need great academic standards and faith in Goddess Gabija as well.
+QUEST_20150317_000649	For this, you will need great academic standards as well as faith in Goddess Gabija.
 QUEST_20150317_000650	Will of All Things
-QUEST_20150317_000651	The simplest forms of all things are Water, Earth, Wind, and Fire, {nl}and these are controlled by the Goddess.
+QUEST_20150317_000651	The base forms of all things are Water, Earth, Wind, and Fire, {nl}and these are controlled by the Goddess.
 QUEST_20150317_000652	Elementary Fire Magic
-QUEST_20150317_000653	1. Dried environment {nl}2. Something combustible. {nl}3. Enough Mana {nl}4. The blessing of Goddess Gabija.
+QUEST_20150317_000653	1. Dried environment{nl}2. Something combustible. {nl}3. Enough mana. {nl}4. The blessing of Goddess Gabiya.
```
Comment (terrapin47) [inline QUEST.tsv]: I believe "Gabija" is correct here.  Referencing the wiki -> [Gabija](https://github.com/ttgmichael/EnglishTranslation/wiki/2.8.%20Names)

File: QUEST.tsv [hunk 10]
```diff
@@ -522,210 +522,210 @@ QUEST_20150317_000521	Seems to be asleep.
...
+QUEST_20150317_000680	Agailla Flurry called Gabija to this tower, so this barrier is not meant to lock Gabija in. {nl}To find out the reason for this, we first have to understand the composition of the barrier.
+QUEST_20150317_000681	After a long investigation, we've found that this barrier was {nl}not only created by the power of Helgasercle, but also with other kinds of mana.
+QUEST_20150317_000682	We've also found that mana was composed by aligning the foundation stones in a certain position.
+QUEST_20150317_000683	We were able to retrieve some of those foundation stones from various areas in the kingdom. {nl}The new information that we found here with them was astonishing.
+QUEST_20150317_000684	The mana provided to use barrier magic wasn't voluntary. {nl}We theorize that the levels of the demon lords were divided and sealed into the foundation stones and aligned into a position.
+QUEST_20150317_000685	The key to unlock this barrier is in the foundation. {nl}We'll continue to find the answer after we block Helgasercle's invasion.
```
Comment (terrapin47) [inline QUEST.tsv]: Changing the second sentence changes the meaning from "surely finding an answer after completing a task" to "continuing to find more information after completing a task".  If that is correct considering the context of the situation ( it looks like it is ), then I'd just say that the wording should be changed a little.
"We'll continue to find more answers"
or
"We'll continue to search for the answer"

File: QUEST.tsv [hunk 11]
```diff
@@ -773,101 +773,101 @@ QUEST_20150317_000772	There are some old pots among the new pots. {nl}Which one
...
+QUEST_20150317_000792	It may be better to guard the seal. {nl}But I believe finding all the lost sisters is the solution to all this.
 QUEST_20150317_000793	I wish you well… {nl}And may you and this land be blessed.
 QUEST_20150317_000794	Right. I think so too. {nl}The monsters are definitely preying on our honey. {nl}But it is better to have evidence on hand, so bring a piece of beehive {nl}from Saltus Bee Farm.
 QUEST_20150317_000795	Coward
-QUEST_20150317_000796	They call me a coward, but I can't do anything. {nl}Because I'm really scared.
-QUEST_20150317_000797	You're the guy the bees were whispering about. {nl}I am the Goddess of Fate, Austeja. 
-QUEST_20150317_000798	The seal of this farmland has stabilized, thanks to you. {nl}But the other Revelators have lost themselves due to the evil force flowing from the Spring Light Woods.{nl}
-QUEST_20150317_000799	I am using all my powers to block the seal from opening any further… {nl}So I can't move away from here, let along save them. {nl}
-QUEST_20150317_000800	Therefore… Please save them and the Spring Light Woods.
+QUEST_20150317_000796	I'm called a coward. I can't do anything. I'm really scared.
```
Comment (terrapin47) [inline QUEST.tsv]: I would still leave in "but" in order to stress the contradiction that this person is making, to being called a coward.

File: QUEST.tsv [hunk 12]
```diff
@@ -773,101 +773,101 @@ QUEST_20150317_000772	There are some old pots among the new pots. {nl}Which one
...
+QUEST_20150317_000833	As you know, the King died on Medis Diena and the fate of other royalty and nobility were no different. {nl}The kingdom is holding together somehow but… now we can only pray.
 QUEST_20150317_000834	Now we can only pray…
-QUEST_20150317_000835	When the spire is bathed with a red light the… door… to fantasy… shall be opened.
+QUEST_20150317_000835	When the spire is bathed with a red light the… door… to fantasy shall be opened.
 QUEST_20150317_000836	Will of King Zachariel
-QUEST_20150317_000837	Dear Revelator.{nl}Listen to the interpretations of the revelation.
-QUEST_20150317_000838	Huh…{nl}Who would've thought I'd be swayed by humans…
-QUEST_20150317_000839	I shall kill you in the most painful way!
+QUEST_20150317_000837	Dear Revelator. {nl}Listen to the interpretations of the revelation.
+QUEST_20150317_000838	Huh… {nl}Who'd've thought I'd be swayed by humans…
```
Comment (terrapin47) [inline QUEST.tsv]: I have to question changing "Who would've" to "Who'd've" here.  It's typically only appropriate if you're quoting someone and reproducing colloquial speech.

**Comment by imcgames** [comment]
Thank you :+1:
